vic/drivers/classic/src/read_veglib.c

```diff
@@ -163,10 +163,10 @@ read_veglib(FILE   *veglib,
             /* Carbon-cycling parameters */
             if (options.VEGLIB_PHOTO) {
                 fscanf(veglib, "%s", tmpstr); /* photosynthetic pathway */
-                if (!strcmp(tmpstr, "C3")) {
+                if (!strcmp(tmpstr, "0") || !strcmp(tmpstr, "C3")) {
                     temp[i].Ctype = PHOTO_C3;
                 }
-                else if (!strcmp(tmpstr, "C4")) {
+                else if (!strcmp(tmpstr, "1") || !strcmp(tmpstr, "C4")) {
```

Comment:
Why don't we figure out how to fix your `tonic` code?  If you can show me what you weren't able to get to work, I'm sure I can help you get it to work. No need to change VIC.